presto-main/src/test/java/com/facebook/presto/operator/aggregation/differentialentropy/TestFixedHistogramJacknifeStateStrategy.java

```diff
@@ -0,0 +1,32 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.facebook.presto.operator.aggregation.differentialentropy;
+
+public class TestFixedHistogramJacknifeStateStrategy
+        extends TestStateStrategy
+{
+    protected static class StrategyCreator
+            implements TestStateStrategy.StrategyCreator
+    {
+        public DifferentialEntropyStateStrategy create(int size)
+        {
+            return new FixedHistogramJacknifeStateStrategy(size, TestStateStrategy.MIN, TestStateStrategy.MAX);
+        }
+    }
+
+    public TestFixedHistogramJacknifeStateStrategy()
+    {
+        super(new StrategyCreator());
```

Comment:
remove StrategyCreator class and use a lambda: `super(size -> new FixedHistogramJacknifeStateStrategy(size, MIN, MAX));`